app/jobs/update_working_days_job.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+class UpdateWorkingDaysJob < ApplicationJob
+  def perform
```

Comment:
I wonder if we should use the fan out sort of pattern for this so as one failure doesn't block the whole lot. Maybe it's not worth the extra complexity here.

Reply:
Yeah, I thought about it and figured it made sense to keep it simple for now and build a more complex solution if we start seeing problems.